src/WebApp/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We do not plan to make this an npm package, therefor, only the `devDependencies` and `scripts` fields are required. In my opinion the rest should be removed because they might become outdated.